test/active_support/cache/redis_store_test.rb

```diff
@@ -176,7 +176,7 @@ def setup
     end
   end
 
-  if RUBY_VERSION.match /1\.9/
+  if RUBY_VERSION.match(/1\.9/)
```

Comment:
This was just to fix a RuboCop syntax warning...I'm totally cool with removing this code at a later date. We don't test against 1.9 anymore.